go.mod

```diff
@@ -4,5 +4,10 @@ go 1.12
 
 require (
 	github.com/Sectorbob/mlab-ns2 v0.0.0-20171030222938-d3aa0c295a8a
+	github.com/mitchellh/go-homedir v1.1.0
```

Comment:
@gssbzn looks like the dep changes were not committed in your base branch

Reply:
I haven't merged so no idea how they got into this PR, also I have no idea why they are not in my pr 🤷‍♂